### PR TITLE
keadm: clean useless file after keadm_e2e test

### PR DESCRIPTION
**Pull request by gccio**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:
1. remove keadm.test and config.json before keadm_e2e.sh exit. keep directory clean.
2. should use `2>&1` to redirect stderr to stdout.

**Which issue(s) this PR fixes**:

```
NONE
```
**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/util/common.go [hunk 1]
```diff
@@ -398,7 +398,7 @@ func killKubeEdgeBinary(proc string) error {
 
 //isKubeEdgeProcessRunning checks if the given process is running or not
 func isKubeEdgeProcessRunning(proc string) (bool, error) {
-	procRunning := fmt.Sprintf("pidof %s 2&>1", proc)
+	procRunning := fmt.Sprintf("pidof %s 2>&1", proc)
```
Comment (daixiang0) [inline keadm/cmd/keadm/app/cmd/util/common.go:401]: Why change it?

Comment (fisherxu) [reply keadm/cmd/keadm/app/cmd/util/common.go:401]: Looks like `2>&1` is right?

Comment (daixiang0) [reply keadm/cmd/keadm/app/cmd/util/common.go:401]: Oh my mistake.

**Pull request merged**